Modules/Filtering/Smoothing/test/itkFFTDiscreteGaussianImageFilterTest.cxx

```diff
@@ -24,30 +24,17 @@
 #include "itkTestingMacros.h"
 #include "itkZeroFluxNeumannBoundaryCondition.h"
 
+template <typename ImageType>
 int
-itkFFTDiscreteGaussianImageFilterTest(int argc, char * argv[])
+itkFFTDiscreteGaussianImageFilterTestProcedure(int argc, char ** argv)
 {
-  if (argc < 5)
-  {
-    std::cerr << "Missing parameters." << std::endl;
-    std::cerr << "Usage:" << std::endl;
-    std::cerr << itkNameOfTestExecutableMacro(argv)
-              << " inputFilename outputFilename sigma kernelError kernelWidth filterDimensionality kernelSource"
-              << std::endl;
-    return EXIT_FAILURE;
-  }
-
-  using ScalarPixelType = float;
-  const size_t ImageDimension = 2;
-  using ImageType = itk::Image<ScalarPixelType, ImageDimension>;
+  float        sigma = (argc > 4) ? std::stof(argv[4]) : 0.0;
```

Comment:
Changes which are not related to increase of test coverage could be squashed into the previous commit.